org.kde.kdenlive.json

```diff
@@ -0,0 +1,466 @@
+{
+    "app-id": "org.kde.kdenlive",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "kdenlive",
+    "rename-icon": "kdenlive",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
+        "--env=FREI0R_PATH=/app/lib/frei0r-1"
+    ],
+    "cleanup": [
+        "/include",
+        "*.la",
+        "*.cmake",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig"
+    ],
+    "modules": [
+        {
+            "name": "gtk2",
+            "config-opts": [
+                "--disable-gtk-doc",
+                "--disable-introspection",
+                "--disable-man",
+                "--disable-static",
+                "--with-xinput=xfree"
+            ],
+            "cleanup": [
+                "/bin",
+                "/share/gtk-2.0",
+                "/lib/gtk-2.0/include"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/gnome/sources/gtk+/2.24/gtk+-2.24.31.tar.xz",
+                    "sha256": "68c1922732c7efc08df4656a5366dcc3afdc8791513400dac276009b40954658"
+                }
+            ]
+        },
+        {
+            "name": "kdecoration",
```

Comment:
Who uses kdecoration? If it's generically required it should possibly go to the Sdk.

Reply:
It was needed for building oxygen.